### PR TITLE
Add KCSNA2021 summit information

### PR DESCRIPTION
**Pull request by mrbobbytables**
This stubs out a good chunk of the initial summit information that we have so far.

### CODE REVIEW

File: content/en/events/kcsna2021/registration.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+### Attend in-person
+
+**NOTE:** The Kubernetes Contributor Summit will follow KubeCon’s
+[Health & Safety Guidelines] requiring **ALL** in-person attendees to be fully
+vaccinated against the COVID-19 virus.
+
+To attend in person you must be a member of one of the Kubernetes GitHub Orgs
+(e.g. [kubernetes] or [kubernetes-sigs]) or a sponsored attendee. If you are an
```
Comment (ameukam) [inline content/en/events/kcsna2021/registration.md:14]: @mrbobbytables Extend to kubernetes-csi and kubernetes-client org ?

Comment (mrbobbytables) [reply content/en/events/kcsna2021/registration.md:14]: I didn't want to add all of the there, but might be worth calling out. Can definitely do it when we publish it with reg links.
All of them are enumerated in the faq though.

**Comment by ameukam** [comment]
/lgtm
/hold
If others want to review it.

File: content/en/events/kcsna2021/faq.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member, can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+October 11th, the Monday before [KubeCon North America].
+
+[KubeCon]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:22]:
```suggestion
[KubeCon North America]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
```

File: content/en/events/kcsna2021/faq.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+
+### Why do I need to register for KubeCon to attend?
+
+Vaccination verification is being handled as part of the [KubeCon registration]
+process. This is the case for all co-located events at KubeCon this year. If
+you cannot attend or register for the event, please reach out to the Summit
+Event team at community@kubernetes.io.
+
+
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:58]:
```suggestion

[KubeCon registration]: /events/kcsna2021/registration/

```

File: content/en/events/kcsna2021/faq.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Till then here are you some additional resources you may find useful:
+- [New Contributor Workshop 2019 - Barcelona](https://www.youtube.com/watch?v=BQ7y2TFOzF4&list=PL69nYSiGNLP2WTJ6P8sQenhf0RY-JqF5L)
+- [New Contributor Workshop 2019 - San Diego](https://www.youtube.com/watch?v=uUJrGwAom-E&list=PL69nYSiGNLP0OWp38tPBc-jSlMmwWr6Ci&index=15)
+- [Kubernetes Contributor Guide](/docs/guide/)
+
+
+### I have a question! How do I contact the event staff?
+
+You can reach out to us directly at community@kubernetes.io or pop into the
+`#contributor-summit` slack channel to ask a question. :)
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:111]: Would it be helpful to link to the Slack channel here, do you think?

File: content/en/events/kcsna2021/faq.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+October 11th, the Monday before [KubeCon North America].
+
+[KubeCon]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.marriott.com/hotels/travel/laxjw-jw-marriott-los-angeles-la-live/" rel="noopener noreferrer" target="_blank">
+JW Marriott LA Live</a>, a 5 minute walk away from the LA Convention Center. See
+the [location] section for more information.
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:29]:
```suggestion
the [Location & Venue] section for more information.
```

File: content/en/events/kcsna2021/faq.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+[location]: /events/kcsna2021/location/
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:31]:
```suggestion
[Location & Venue]: /events/kcsna2021/location/
```

File: content/en/events/kcsna2021/faq.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+---
+
+- [When is the summit taking place?](#when-is-the-summit-taking-place)
+- [Where is the summit located?](#where-is-the-summit-located)
+- [Will I be able to attend remotely?](#will-i-be-able-to-attend-remotely)
+- [Why is vaccination required to attend in-person?](#why-is-vaccination-required-to-attend-in-person)
+- [Why do I need to register for KubeCon to attend?](#why-do-i-need-to-register-for-kubecon-to-attend)
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member, can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:14]:
```suggestion
- [I am not an Org member. Can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
```

File: content/en/events/kcsna2021/faq.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+**NOTE:** This applied to attending in person only. Anyone may register and
+attend remotely.
+
+If you are a SIG lead or subproject owner and there is someone that is not yet
+an org member or from another project that might take part of a panel,
+SIG face-to-face meeting etc, you can sponsor them to attend the summit.
+Please reach out to community@kubernetes.io to make arrangements.
+
+
+### I am not an Org member, can I still attend?
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:88]:
```suggestion
### I am not an Org member. Can I still attend?
```

File: content/en/events/kcsna2021/faq.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+### Will I be able to attend remotely?
+
+Yes! We are planning a full hybrid event that **should** allow those
+participating remotely to interact and work with contributors on-site. We will
+post more details on how this will work closer to the event.
+
+
+### Why is vaccination required to attend in-person?
+
+As a co-located event the summit is required to adhere to KubeCon’s
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:43]:
```suggestion
As a co-located event, the summit is required to adhere to KubeCon’s
```

File: content/en/events/kcsna2021/faq.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+and on-site staffing availability. 
+
+
+### Will there be a New Contributor Workshop?
+
+No, with uncertain attendance and staffing, we made this decision to ensure we
+can put on an event to the best of our ability. For those that are interested
+in becoming a contributor SIG ContribEx plans to launch a course later this year.
+
+Till then here are you some additional resources you may find useful:
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:102]:
```suggestion
Until then, here are you some additional resources you may find useful:
```

File: content/en/events/kcsna2021/faq.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+[Kubernetes Org Members](#why-is-vaccination-required-to-attend-in-person) and
+[sponsored attendees](#what-is-a-sponsored-attendee) due to uncertain attendance
+and on-site staffing availability. 
+
+
+### Will there be a New Contributor Workshop?
+
+No, with uncertain attendance and staffing, we made this decision to ensure we
+can put on an event to the best of our ability. For those that are interested
+in becoming a contributor SIG ContribEx plans to launch a course later this year.
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:100]:
```suggestion
in becoming a contributor, SIG ContribEx plans to launch a course later this year.
```

File: content/en/events/kcsna2021/faq.md [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+
+
+### Why is vaccination required to attend in-person?
+
+As a co-located event the summit is required to adhere to KubeCon’s
+[Health & Safety Guidelines]. There may be additional restrictions that develop
+closer to the event, and we will update the event accordingly to follow venue,
+municipality and CDC guidelines at a minimum.
+
+[Health & Safety Guidelines]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/health-and-safety/
```
Comment (nimbinatus) [inline content/en/events/kcsna2021/faq.md:48]: Should this pop to a new tab like on the location page?

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Some patches and a question or two.

File: content/en/events/kcsna2021/faq.md (hunk 12 above)
Comment (mrbobbytables) [reply content/en/events/kcsna2021/faq.md:48]: yeah, I'll take care of that 👍

**Pull request merged**